lib/Parser/Parse.cpp

```diff
@@ -4650,7 +4668,19 @@ ParseNodePtr Parser::ParseMemberList(LPCOLESTR pNameHint, uint32* pNameHintLengt
                 pnodeArg = CreateBinNode(knopMember, pnodeName, pnodeFnc);
             }
         }
-        else if (nullptr != pidHint) //Its either tkID/tkStrCon/tkFloatCon/tkIntCon
+		else if (useSpread)
+		{
+			pnodeSpread = ParseExpr<buildAST>(koplCma, nullptr, TRUE, /* fAllowEllipsis */ TRUE);
+			if (buildAST)
+			{
+				//if (pnodeArg->nop == knopEllipsis)
```

Comment:
What's this commented code for? Just for experimenting while you were working on the feature?

Reply:
yeah, let me get rid of that